api/api.yml

```diff
@@ -89,20 +103,30 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ComputationDetails'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '404':
+          $ref: '#/components/responses/EntityNotFoundError'
     delete:
       tags: 
       - launcher
       summary: Stop computation
       operationId: stopComputation
+      security:
+      - OAuth2: [appuser]
       parameters:
       - in: path
         name: id
         schema:
           type: string
         required: true
       responses:
-        '200':
+        '202':
```

Comment:
Co to znaczy? Że OK, ale może się nie stać?

Reply:
Że po pierwsze OK musi mieć body.
Po drugie 200 jest zwracane po fakcie.
A 202 przed faktem, czyli takie "Przyjęte do realizacji".